frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1223,24 +1239,21 @@ function ContextMenuInner(propsWithInputRef: Props) {
           formatNumberToVolume(volumeInNm3),
           `${boundingBoxTopLeftString}, ${boundingBoxSizeString}`,
         ];
+      } catch (_error) {
+        const notFetchedMessage = "could not be fetched";
```

Comment:
Would a capital C ("Could not ...") look better in the UI?

Reply:
Personally I think lower-case looks better, but let me know if you disagree!

Reply:
![image](https://github.com/scalableminds/webknossos/assets/57364771/5e5a8db9-9ca7-44eb-b2c6-e9c8a5928f18)
![image](https://github.com/scalableminds/webknossos/assets/57364771/f842bef2-f151-4752-9341-5f076c2bde2e)
for reference

Reply:
okay, then let's stick to lower case :+1: